templates/repo/scripts/component.mk

```diff
@@ -8,41 +8,71 @@ include $(SELF_DIR)/common.mk
 all:
 .PHONY: all
 
+setup:
+	$(MAKE) -C $(REPO_ROOT) setup
+
 check: lint check-plan
 .PHONY: check
 
 fmt: terraform
-	$(sh_command) -c 'for f in $(TF); do printf .; terraform fmt $$f; done'; \
-	echo
+	@printf "fmt: ";
+	@for f in $(TF); do printf .; terraform fmt $$f; done
+	@echo
 .PHONY: fmt
 
 lint: terraform-validate lint-terraform-fmt lint-tflint
 .PHONY: lint
 
 lint-tflint: init
-	if (( $$TFLINT_ENABLED )); then \
-    $(sh_command) -c 'tflint' || exit $$?; \
-	fi
+	@printf "tflint: "
+ifeq ($(TFLINT_ENABLED),1)
+	@tflint || exit $$?;
+else
+	@echo "disabled"
+endif
 .PHONY: lint-tflint
 
 terraform-validate: terraform init
-	$(sh_command) -c 'terraform validate -check-variables=true'
+	@$(terraform_command) validate -check-variables=true
 .PHONY: terraform-validate
 
 lint-terraform-fmt: terraform
-	$(sh_command) -c 'for f in $(TF); do printf .; terraform fmt --check=true --diff=true $$f || exit $$? ; done'
+	@printf "fmt check: "
+	@for f in $(TF); do \
+		printf . \
+		$(terraform_command) fmt --check=true --diff=true $$f || exit $$? ; \
+	done
+	@echo
 .PHONY: lint-terraform-fmt
 
-get: terraform
-	$(terraform_command) get --update=true
-.PHONY: get
-
-plan: terraform fmt get init
-	$(terraform_command) plan
+ifeq ($(MODE),local)
+plan: init fmt
+	@$(terraform_command) plan
+else ifeq ($(MODE),atlantis)
+plan: init lint
+	@$(terraform_command) plan -input=false -no-color -out $(PLANFILE) | scenery
```

Comment:
what's `scenery`?

Reply:
A terraform output prettifier written in go https://github.com/dmlittle/scenery

I have it baked into the atlantis image. Considering using fogg to install it for local mode too, but I want to see what happens with 0.12 output (it is supposedly better).